buildenv/jenkins/JenkinsfileBase

```diff
@@ -848,6 +846,8 @@ def post(output_name) {
 		}
 		String failedTests = addFailedTestsGrinderLink()
 		triggerRerunJob(failedTests)
+
+		junit allowEmptyResults: true, keepLongStdio: true, testResults: '**/work/**/*.jtr.xml, **/result/**/*.jtr.xml, **/junitreports/**/*.xml, **/external_test_reports/**/*.xml'
```

Comment:
I think we should use try catch instead. Other steps can fail as well. If they fail, we do not want junit report to be missing.

Reply:
Other steps failures means bugs we need to fix. This failure might be the issue with the junit plugin, which we don't have control.

Reply:
> Other steps failures mean bugs we need to fix.

That is not always the case. For example, we could have a network issue that results in a failure to archive the artifacts.

Reply:
The Jenkins Junit Plugin itself doesn't generate/throw exception, it only prepare/display unit tests report. Not sure if the try catch can catch the exception or not. Updated and see how it works.